metrics/Proposals.md

```diff
@@ -0,0 +1,124 @@
+### Parameters
+
+Mandatory:
+
+* Period of time. Start and finish date of the period. Default: forever.
+
+    Period during which proposals are considered.
+    
+* Criteria for source code. Algorithm. Default: all files are source code.
+
+    If we are focused on source code, we need a criteria for deciding
+    whether a file is a part of the source code or not.
+    
+### Aggregators
+
+Usual aggregators are:
+
+* Count. Total number of proposals during the period.
+
+## Specific description: GitHub
+
+In the case of GitHub, a proposal is defined as a "pull request",
+as long as it proposes changes to source code files.
+
+The date of the proposal can be defined (for considering it in a period or not)
+as the date in which the pull request was submitted.
+
+### GitHub parameters
+
+None.
+
+## Specific description: GitLab
+
+In the case of GitLab, a proposal is defined as a "merge request",
+as long as it proposes changes to source code files.
+
+The date of the proposal can be defined (for considering it in a period or not)
+as the date in which the merge request was submitted.
+
+### GitLab parameters
+
+None.
+
+## Specific description: Gerrit
+
+In the case of Gerrit, a proposal is defined as a "code review",
+or in some contexts, a "changeset",
+as long as it proposes changes to source code files.
+
+The date of the proposal can be defined (for considering it in a period or not)
+as the date in which the code review was started by submitting a
+patchset for review.
+
+### Gerrit parameters
+
+None.
+
+
+## Use Cases
+
+* Volume of changes proposed to a project.
+
+    Proposals are a proxy for the activity in a project.
+    By counting proposals to change code in the set of repositories corresponding
+    to a project, you can have an idea of the overall activity in
+    proposing changes to that project.
+    Of course, this metric is not the only one that should be
+    used to track volume of coding activity.
+
+
+## Filters
+
+Usual filters and bucketing are:
+
+* By actors (submitter, reviewer, merger). Requires actor merging
+(merging ids corresponding to the same author).
+
+* By groups of actors (employer, gender... for each of the actors).
+Requires actor grouping, and likely, actor merging.
+
+## Visualizations
+
+Some useful visualizations are:
+
+* Count per month over time
+* Count per group over time
+
+These could be represented as bar charts, with time running in the X axis.
+Each bar would represent proposals to change the code
+during a certain period (eg, a month).
+
+## Reference Implementation
+
+[ To be done. ]
+
+## Known Implementations
+
+* [Grimoirelab](https://chaoss.github.io/grimoirelab). Enriched index for
+GitHub Pull Requests, GitLab Merge Requests,
+and Gerrit repositories is composed of one item per proposal,
+which makes it basically correspond to this metric when counted.
+The GitHub Pull Requests, GitLab Merge Requests, and Gerrit summary panels,
+available out of the box, provide exactly that.
+
```

Comment:
There is also an existing implementation for these in Augur: 
[Augur](http://dev.augurlabs.io/metrics_status)

Reply:
I see two entries for a metric "Pull_Requests_Open", but both seem to link to the [same definition](https://github.com/augurlabs/metrics/blob/wg-gmd/activity-metrics/pull-requests-open.md), which corresponds to the SQL in the comment above. As I understand, this corresponds to pull requests open as of now (well, as of the last update in the GHTorrent database), not to this metric.

Maybe I'm missing another metric which corresponds to this one?